tests/ballerina-unit-test/src/test/resources/test-src/taintchecking/propagation/iterable-within-iterable-negative.bal

```diff
@@ -0,0 +1,38 @@
+json globalVar;
+
+function main (string... args) {
+    taintedJsonReturn().address_components
+    .filter((json comp) => boolean {return comp.types.filter((json compType) => boolean {secureFunction(taintedReturn(), taintedReturn()); return compType.toString() == "street_number";}).count() > 0; })
```

Comment:
Shall we reformat this to have 120 width?

Reply:
+1